nucliadb/nucliadb/reader/api/v1/services.py

```diff
@@ -113,8 +110,14 @@ async def get_entity(request: Request, kbid: str, group: str) -> EntitiesGroup:
             )
         )
```

Comment:
I guess you missed removing the second assignation to `response`. Also, do we need to filter the "" key from the entities key like above?